subsys/bluetooth/host/l2cap.c

```diff
@@ -83,8 +83,14 @@ struct bt_l2cap {
 	struct bt_l2cap_le_chan	chan;
 };
 
+static struct bt_l2cap_cb *l2cap_callbacks;
 static struct bt_l2cap bt_l2cap_pool[CONFIG_BT_MAX_CONN];
 
+void register_l2cap_callbacks(struct bt_l2cap_cb *cb)
```

Comment:
Even though this is internal, please use proper namespacing (it’s also consistent with the fact that the parameter type is properly namespaced). Eg `bt_l2cap_register_ecred_cb`